tests/testthat/test-NetworkMeasures.R

```diff
@@ -0,0 +1,248 @@
+context("Correctness of Network Measures")
+
+##### TEST SET UP #####
+
+rm(list = ls())
+# Configure logger (suppress all logs in testing)
+# expect_silents only work with this logger turned off; only alerts with warnings
+loggerOptions <- futile.logger::logger.options()
+if (!identical(loggerOptions, list())){
+    origLogThreshold <- loggerOptions[[1]][['threshold']]
+} else {
+    origLogThreshold <- futile.logger::INFO
+}
+futile.logger::flog.threshold(0)
```

Comment:
for another day...now that you taught me that `setupTestEnv` thing...could we put a `silence_logger` function in that that we reused in every test? Or even just run this code there?

I'll make an issue and we can discuss there

Reply:
created #170

Reply:
Honestly, I just copy-pasta'd the header and footer code from the other test files.  Open to changes after this PR.